dwst/scripts/controller/error.js

```diff
@@ -14,6 +14,32 @@
 import errors from '../lib/errors.js';
 const {NoConnection, AlreadyConnected, SocketError, InvalidParticles, InvalidArgument, InvalidCombination, UnknownCommand, UnknownInstruction, UnknownHelpPage, UnknownText, UnknownBinary} = errors;
 
+function commaCommaOr(stringList) {
```

Comment:
I'm fine having this here as long as it is not needed elsewhere. We could move it to lib.utils if we needed it in multiple places.